terraform/shared/scripts/client.sh

```diff
@@ -14,8 +14,8 @@ HOME_DIR=ubuntu
 # Wait for network
 sleep 15
 
-# IP_ADDRESS=$(curl http://instance-data/latest/meta-data/local-ipv4)
-IP_ADDRESS="$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')"
+IP_ADDRESS=$(curl http://instance-data/latest/meta-data/local-ipv4)
+# IP_ADDRESS="$(/sbin/ifconfig eth0 | grep 'inet addr:' | cut -d: -f2 | awk '{ print $1}')"
```

Comment:
Does the `http://instance-data` approach work in Azure as well?

Reply:
ugh, i'm not sure. i know that the approach that was in here before didn't work either, because it assumed `eth0`. this needs to be rewritten one way or the other.

Reply:
Interesting - what is the network interface name were you getting?